metricbeat/helper/metricset.go

```diff
@@ -99,3 +102,24 @@ func (m *MetricSet) createEvent(event common.MapStr) common.MapStr {
 
 	return event
 }
+
+func applySelector(event common.MapStr, selectors []string) common.MapStr {
+
+	// Not selectors set means return full events
+	if len(selectors) == 0 {
+		return event
+	}
+
+	newEvent := common.MapStr{}
+
+	for _, selector := range selectors {
+		logp.Debug("Selector", "selector, %s", selector)
```

Comment:
@andrewkroh yes, I actually didn't event wanted to have this in the code. but I will change it and make it more readable.